cheatsheets/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -191,7 +193,7 @@ object-src 'none'; base-uri 'none';
 
 ### Refactoring inline code
 
-By default CSP disables any unsigned JavaScript code placed inline in the HTML source, such as this:
+When `default-src` or `script-src*` directives are active, CSP by default disables any unsigned JavaScript code placed inline in the HTML source, such as this:
```

Comment:
unsigned? I do not believe digital signatures are a part of CSP

Reply:
Suggest changing this to:

When `default-src` or `script-src*` directives are active, CSP by default disables any JavaScript code placed inline in the HTML source, such as this:

And can you also please remove the other use of the term signed in this document?

Reply:
I believe this references two main ways of "signing" code with CSP:
1. `nonce` 
2. SHA hashes (as describes right after.

We could change to 
```suggestion
When `default-src` or `script-src*` directives are active, CSP by default disables any inline JavaScript code, specified as script-elements or script-attributes, such as this:
```

Reply:
signed in the wrong term. Here is the exact text from the CSP 2 standard to consider.

The script-src directive lets developers specify exactly which script elements on a page were intentionally included for execution. Ideally, developers would avoid inline script entirely and whitelist scripts by URL. However, in some cases, removing inline scripts can be difficult or impossible. For those cases, developers can whitelist scripts using a randomly generated nonce.

and

The script-src directive lets developers whitelist a particular inline script by specifying its hash as an allowed source of script.

Reply:
And here is similar text from the CSP 3 standard to consider

The script-src directive governs four things:

Script requests MUST pass through §4.1.3 Should request be blocked by Content Security Policy?.

Script responses MUST pass through §4.1.4 Should response to request be blocked by Content Security Policy?.

Inline script blocks MUST pass through §4.2.3 Should element’s inline type behavior be blocked by Content Security Policy?. Their behavior will be blocked unless every policy allows inline script, either implicitly by not specifying a script-src (or default-src) directive, or explicitly, by whitelisting "unsafe-inline", a nonce-source or a hash-source that matches the inline block.

The following JavaScript execution sinks are gated on the "unsafe-eval" source expression:

eval()

Function()

setTimeout() with an initial argument which is not callable.

setInterval() with an initial argument which is not callable.

Reply:
Sincerely, there are zero mentions of digial signatures or similar in the CSP standard. =)